packages/backend/knexfile.js

```diff
@@ -3,11 +3,11 @@ module.exports = {
     client: 'postgresql',
     // connection: 'postgres://postgres:password@postgres/testnet',
     connection: {
-      host: 'postgres',
+      host: '0.0.0.0',
       database: 'testnet',
       user: 'postgres',
       password: 'password',
-      port: 5432
+      port: 5433
```

Comment:
If this is 5432, knex commands won't work as postgres server is working at 5433

Reply:
Postgres server is actually working on 5432 on our docker container. Just that it's port is mapped to 5433 on the host machine. 

I think Rico is not developing using the docker containers, he is using his host machine to connect and that is why he uses 5433. But on the main branch, it should be 5432, and the host should be postgres.
That is so that the inner docker network's dns can resolve when containers are communicating with each other.

When you want to use knex commands directly from the host machine, yes you should set it up as rico did. Port 5433 and localhost as host. 
But since the main branch is where we expect the docker environment to run properly, not based on how each of us has his local dev setup, this PR should revert these changes.

Reply:
Well, it's clear. But then wouldn't it be better to update every port into 5432 and remove 5433 in our codebase?

Reply:
> Well, it's clear. But then wouldn't it be better to update every port into 5432 and remove 5433 in our codebase?

+1

Reply:
Conflicts with rafiki prevent this from happening now. 
we will be able to do it once we get rafiki to be a npm package instead of having the whole project cloned into our repo.